app/assets/stylesheets/upvote/posts.css.scss

```diff
@@ -6,6 +6,25 @@ h3.no-posts {
 $vote_container_size: 45px;
 $vote_container_margin: 12px;
 
+div.date {
+  @include clearfix;
+  margin-bottom: 12px;
+  .header {
+    margin-bottom: 12px;
+    .permalink {
+      margin-left: 12px;
```

Comment:
Properties should be ordered color, display, font-size, line-height, margin-left, text-transform